src/bundlers/utils.ts

```diff
@@ -87,3 +87,80 @@ export async function bundle(
     debugLogger.debug(`The bundling process finished successfully for file ${element.path}.`);
     return output;
 }
+
+export function getLocalFunctionWrapperCode(handler: string, entry: string, language: string) {
+    switch (language) {
+        case "js":
+        case "ts":
+            return `import { ${handler} as userHandler } from "./${entry}";
+
+import http from "http";
+
+const port = process.argv[2];
+
+const server = http.createServer((req, res) => {
+  if (req.method === 'POST') {
+    let body = '';
+    req.on('data', chunk => {
+      body += chunk.toString();
+    });
+    req.on('end', () => {
+      res.writeHead(200, {'Content-Type': 'text/plain'});
+      const jsonParsedBody = JSON.parse(body);
+
+      userHandler(jsonParsedBody).then((response) => {
+        res.end(JSON.stringify(response));
+    })
+    });
+  } else {
+    res.writeHead(404, {'Content-Type': 'text/plain'});
+    res.end('404 Not Found');
+  }
+});
+
+server.listen(port, () => {
+});
+`;
+        case "python":
+            return `
+import sys
+import json
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from ${entry.split(".")[0]} import ${handler} as userHandler
+
+class RequestHandler(BaseHTTPRequestHandler):
+
+    def do_POST(self):
+        content_length = int(self.headers['Content-Length'])
+        post_data = self.rfile.read(content_length)
+        self.send_response(200)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+
+        try:
+            jsonParsedBody = json.loads(post_data)
+            response = userHandler(jsonParsedBody)
+            self.wfile.write(json.dumps(response).encode('utf-8'))
+            sys.stdout.flush()
+        except Exception as e:
+            self.send_response(500)
+            self.wfile.write(f"Error: {str(e)}".encode('utf-8'))
+            sys.stdout.flush()
+
+def run():
+    port = int(sys.argv[1])
+    server_address = ('', port)
+    httpd = HTTPServer(server_address, RequestHandler)
+    try:
+        httpd.serve_forever()
+    except KeyboardInterrupt:
+        sys.stdout.flush()  
+        httpd.server_close()
+
+if __name__ == "__main__":
+    run()
+`;
+        default:
+            return ``;
```

Comment:
This default might hide a subtle error because it's silent - we should never reach this in silence :))

On default I would throw an error:
```
throw new Error(`${language} is not supported yet for testing locally`);
```